tethys_apps/migrations/0004_customsetting_default.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.8 on 2019-05-09 19:49
```

Comment:
there is already a `0004_...` migration in `tethys_apps/migrations`, so we should rename this to `0005_...`.